pkg/apis/openebs.io/v1alpha1/cstor_pool.go

```diff
@@ -37,10 +37,18 @@ type CStorPool struct {
 
 // CStorPoolSpec is the spec listing fields for a CStorPool resource.
 type CStorPoolSpec struct {
-	Disks    DiskAttr      `json:"disks"`
+	Group    []DiskGroup      `json:"group"`
```

Comment:
Can we have comments on each property of this schema?
We can comment on the properties that remains unchanged as well.

Reply:
It is a WIP PR. I will add them.